### ISSUE TITLE
Missing Pages menu button for visitors on community.phpfox.com

### ISSUE THREAD

**Issue by PhpFoxJohnJr** [moxi9/phpfox#690]
Ok, I have two issues.

First when you are a visitor even on the community.phpfox.com site you will see the hard code Pages menu does not show...see attached photo.

Also, though it looks like it is solved on the demo site community.phpfox.com.  I have installed 4.09 and I am using the wisteria theme and the menu for Pages does not show up on the mobile version whether it is apple or android.

![phpfoxerrorpages](https://cloud.githubusercontent.com/assets/14913734/10861804/74f2c0b6-7f5b-11e5-8bc9-295add8530c9.jpg)

**Comment by Data66PF** [comment]
Please post only one issue per bug report. Note that this area is only for the core product but since we don't have a separate section for wisteria, you could post that issue in another bug report but please search first in case it was reported.

**Comment by PhpFoxJohnJr** [comment]
I know I wrote two issues on top but I also wrote this...

"Also, though it looks like it is solved on the demo"

So I don't know if it is another issue or part of the same issue since community.phpfox.com isn't running wisteria so if it is a coding problem in the core then it  shouldn't matter what the theme is...right?

**Comment by PhpFoxJohnJr** [comment]
I guessed I got confused in regards to the wisteria theme maybe being the problem because I thought the mobile version of community.phpfox.com worked but it doesn't.  As I just reviewed it again and the issue is that the menu for PAGES does not show up for the website on a pc or the mobile version when you are not logged in.  Has nothing to do with Wisteria...it is just a bug that will probably be fixed for the mobile version when the pc version is fixed...sorry for the confusion.
